app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -41,13 +41,7 @@
             this.formattedBeleidsdomein
           }}</div>
       </div>
-      {{#unless @editing}}
```

Comment:
ik vind het nu precies wel wat cluttered als al die tekst voor het editten altijd zichtbaar is

Reply:
sure but otherwise you're clicking a million times on 'edit', first to go to the person view, then to get to the mandataris, then to toggle editing view and finally to choose what kind of edit you'd like to do... Feels clicky man

Reply:
yeah you're also right, then leave it till we get feedback from the designer 😉